### DISCUSSION TITLE
Possibility to mark room as unread

### DISCUSSION THREAD

**Discussion by brandsimon** [element-hq/element-meta#816] [Ideas]
#### What would you like to do?

I want to mark a chat (I already read) as unread.

#### Why would you like to do it?

So I remember to answer someone or act on something, which I don't want to do now.

#### How would you like to achieve it?

Room-overview:
I want a button in the menu, when long-pressing on a room.

In a room:
I want a button when opening the three-dot menu

### Have you considered any alternatives?

No

### Are you willing to provide a PR?

No

**Comment by immanuelfodor** [comment]
It would be great to mark a room unread by marking a message unread, so any further messages are marked as unread from that point if I mark an older message as unread. Can be useful with rooms of many messages previously unread when opened the room, so to mark from where to follow up later. RocketChat has the same functionality to mark any message a room as unread (and all following messages as well).

**Comment by mnaturel** [comment]
Hi @brandsimon, thanks for your suggestion. I moved the issue to the meta repository and converted it to a discussion as it affects all Element clients.

**Comment by vitriolix** [comment]
This is pretty core part of my Slack workflow that helps me not lose my place in the room history and to remind me to come back to reading where I left off.  

For instance: a room with 100 messages might be too much for me to catch up on all at once.  I read down 20 messages and want to come back later, so I mark message 21 as unread and move onto other rooms as needed knowing i can come back to where I left off
